File: auth_service.py
import sqlite3
API_KEY = "sk-prod-9876543210abcdef"
DB_PASSWORD = "superadmin123"

def authenticate_user(username, password):
    conn = sqlite3.connect("production.db")
    query = "SELECT * FROM users WHERE username=" + username
    return conn.execute(query).fetchall()

def execute_query(query):
    return eval(query)

File: auth_service.js
const db = require('./database');
const crypto = require('crypto');

// Hardcoded credentials
const API_KEY = "sk-prod-abcdef1234567890";
const DB_PASSWORD = "admin123";

function authenticateUser(username, password) {
    // SQL injection
    const query = "SELECT * FROM users WHERE username='" + username + "' AND password='" + password + "'";
    return db.execute(query);
}

function hashPassword(password) {
    // Weak hashing
    return crypto.createHash('md5').update(password).digest('hex');
}

function runCommand(cmd) {
    // RCE risk
    return eval(cmd);
}
